recipe/meta.yaml

```diff
@@ -1,28 +1,28 @@
 {% set name = "Flask-WTF" %}
-{% set version = "0.15.1" %}
+{% set version = "1.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: ff177185f891302dc253437fe63081e7a46a4e99aca61dfe086fb23e54fff2dc
+  url: https://files.pythonhosted.org/packages/d9/38/d4798dd05be711d666e1befb08b1d3db57bd146d1a14d15657f60c88b446/{{ name }}-{{ version }}.tar.gz
+  sha256: 34fe5c6fee0f69b50e30f81a3b7ea16aa1492a771fe9ad0974d164610c09a6c9
 
 build:
-  noarch: python
+  skip: True # [py<37]
```

Comment:
please make sure there are two spaces before the '#' sign, and one before the selector